website/docs/r/api_management.html.markdown

```diff
@@ -299,6 +301,12 @@ A `sign_up` block supports the following:
 
 ---
 
+A `tenant_access` block supports the following:
+
+* `enabled` - (Required) Should the access to the management api be enabled?
+
```

Comment:
we are missing the rest of the blocks propeties here?

Reply:
The missing properties are only exported and cannot be set. I added them to the 'Attribute reference' part of the docs.